src/support/index.coffee

```diff
@@ -105,7 +105,10 @@ module.exports = ->
     argv['prevent-browser-reload']?
 
   terminateDriver = =>
-    @driver.close()
-    @driver.quit()
+    if @driver?
+      @driver.close()
+      @driver.quit()
+    else
+      $.defer()
```

Comment:
yeah I think this the idea behind this is a good PR. mind just updating a touch and we can get this merged in?